src/Umbraco.Cms.Integrations.SEO.Semrush/Api/Management/Controllers/Token/GetAccessTokenController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Cms.Integrations.SEO.Semrush.Configuration;
+using Umbraco.Cms.Integrations.SEO.Semrush.Models.Dtos;
+using Umbraco.Cms.Integrations.SEO.Semrush.Services;
+
+namespace Umbraco.Cms.Integrations.SEO.Semrush.Api.Management.Controllers.Token
+{
+    public class GetAccessTokenController : TokenControllerBase
+    {
+        public GetAccessTokenController(IOptions<SemrushSettings> options, IWebHostEnvironment webHostEnvironment, ISemrushTokenService semrushTokenService, ICacheHelper cacheHelper, TokenBuilder tokenBuilder, SemrushComposer.AuthorizationImplementationFactory authorizationImplementationFactory) : base(options, webHostEnvironment, semrushTokenService, cacheHelper, tokenBuilder, authorizationImplementationFactory)
+        {
+        }
+
+        [HttpPost("get")]
+        [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
```

Comment:
Could consider also a `500-Internal Server Error` as the service method could encounter an error. So you could validate the returned string if it contains "Error" and handle accordingly.